e2e/wdio/core/tests/dynamic-page.e2e-spec.ts

```diff
@@ -0,0 +1,335 @@
+            checkPageClosed(saveButton);
+        });
+
+        it('should check close page by clicking close button', () => {
+            openPage(tabsExample);
+            checkPageClosed(closeButton);
+        });
+
+        it('should check pining area', () => {
+            checkPiningArea(tabsExample);
+        });
+
+        it('should check collapsing area', () => {
+            checkCollapsingArea(tabsExample);
+        });
+
+        it('should check selecting tabs', () => {
+            click(tabsExample + button);
+            scrollIntoView(tabsContent, 1);
+            expect(getElementClass(tab, 1)).toContain('is-selected', 'tab is not highlited as selected after scroll to content');
+            expect(getElementClass(tab, 0)).not.toContain('is-selected', 'tab is selected, but should not');
+            expect(getElementClass(tab, 2)).not.toContain('is-selected', 'tab is selected, but should not');
+        });
+
+        it('should check links clickable', () => {
+            checkLinksClickable(tabsExample);
+        });
+
+        it('should check navigate by link', () => {
+            checkUrlNavigation(tabsExample);
+        });
+
+    });
+
+    describe('Tests for default example', () => {
+
+        it('should check close page by clicking exit button', () => {
+            openPage(defaultExample);
+            checkPageClosed(exitButton);
+        });
+
+        it('should check close page by clicking cancel button', () => {
+            openPage(defaultExample);
+            checkPageClosed(cancelButton);
+        });
+
+        it('should check close page by clicking reject button', () => {
+            openPage(defaultExample);
+            checkPageClosed(rejectButton);
+        });
+
+        it('should check close page by clicking accept button', () => {
+            openPage(defaultExample);
+            checkPageClosed(acceptButton);
+        });
+
+        it('should check close page by clicking save button', () => {
+            openPage(defaultExample);
+            checkPageClosed(saveButton);
+        });
+
+        it('should check close page by clicking close button', () => {
+            openPage(defaultExample);
+            checkPageClosed(closeButton);
+        });
+
+        it('should check links clickable', () => {
+            checkLinksClickable(defaultExample);
+        });
+
+        it('should check navigate by link', () => {
+            checkUrlNavigation(defaultExample);
+        });
+
+    });
+
+    it('should check RTL and LTR orientation', () => {
+        dynamicPagePage.checkRtlSwitch();
+    });
+
+    function checkPageClosed(button: string): void {
+        click(button);
+        expect(doesItExist(dynamicPageContent)).toBe(false, 'dynamic page is not closed');
+    }
+
+    function openPage(section: string): void {
+        click(section + button);
+        expect(isElementDisplayed(dynamicPageContent)).toBe(true, 'dynamic page is not opened')
```

Comment:
you can just do waitForElDisplayed(dynamicPageContent); because this method is to open page, not check that the page is opened, no assertion needed. your choice